modin/pandas/api/extensions/__init__.py

```diff
@@ -0,0 +1,37 @@
+# Licensed to Modin Development Team under one or more contributor license agreements.
+# See the NOTICE file distributed with this work for additional information regarding
+# copyright ownership.  The Modin Development Team licenses this file to you under the
+# Apache License, Version 2.0 (the "License"); you may not use this file except in
+# compliance with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under
+# the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific language
+# governing permissions and limitations under the License.
+
+"""
+Public API for extending modin objects.
+"""
+
+from pandas.core.accessor import _register_accessor
+from pandas.api.extensions import *  # noqa: F401, F403
+
+
+def register_dataframe_accessor(name):
+    from modin.pandas import DataFrame
+
+    return _register_accessor(name, DataFrame)
```

Comment:
In pandas the function looks like:

```python
def register_dataframe_accessor(name):
    from pandas import DataFrame

    return _register_accessor(name, DataFrame)
```

Therefore, when called, it modify pandas objects, not modin. So we can't just import them unchanged.